### PR TITLE
docs: add intentOptions for espresso and screenshotOrientation for xcuitest

### PR DESCRIPTION
**Pull request by KazuCocoa**
## Proposed changes

- https://github.com/appium/WebDriverAgent/pull/277
- https://github.com/appium/appium-espresso-driver/issues/538

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/en/advanced-concepts/settings.md [hunk 1]
```diff
@@ -93,3 +93,4 @@ Settings are implemented via the following API endpoints:
...
 | `dismissAlertButtonSelector` | Allows to customize dismiss alert button selector. It helps you to handle an arbitrary element as dismiss button in `dismiss alert` command. The selector should be a valid [class chain](https://github.com/facebookarchive/WebDriverAgent/wiki/Class-Chain-Queries-Construction-Rules) expression, where the search root is the alert element itself. The default button location algorithm is used if the provided selector is wrong or does not match any element. | e.g., <code>**/XCUIElementTypeButton[\`label CONTAINS[c] 'dismiss'\`]</code> | 1.16.0+ |
+| `screenshotOrientation` | Adjust screenshot orientation for iOS. Appium tries to return a screenshot correctly adjusting the orientation automatically by default, but it sometimes does not work, especially landscape mode. It depends on various factors such as OS versions, model versions and real or simulator devices. This option allows you to adjust the coordinate. Defaults to `auto`.| `auto`, `portrait`, `portraitUpsideDown`, `landscapeRight` | 1.17.0+ |
```
Comment (mykola-mokhnach) [inline docs/en/advanced-concepts/settings.md]: in landscape mode

Comment (mykola-mokhnach) [inline docs/en/advanced-concepts/settings.md]: The actual screenshot orientation depends...

Comment (mykola-mokhnach) [reply docs/en/advanced-concepts/settings.md]: ...and whether this is  a real or simulator devices

Comment (mykola-mokhnach) [reply docs/en/advanced-concepts/settings.md]: ... allows you to enforce the given image orientation

Comment (mykola-mokhnach) [reply docs/en/advanced-concepts/settings.md]: landscapeRight should not be there?

File: docs/en/writing-running-appium/caps.md [hunk 2]
```diff
@@ -167,6 +167,7 @@ These Capabilities are available only on the [Espresso Driver](/docs/en/drivers/
 |`espressoServerLaunchTimeout`|Timeout in milliseconds used to wait for the Espresso server to launch. Defaults to `30000` |e.g., `50000`|
 |`espressoBuildConfig`|Path to the Espresso server build configuration JSON (see below)|e.g., `/projects/myapp-tests/buildconfig.json`|
 |`showGradleLog`|Whether to pipe Gradle build log for the Espresso server to the Appium log. Defaults to `false`|e.g., `true`|
+|`intentOptions`|Intent options which will be used to start the application under test. It can set intent options such as `action`, `categories` and `flags` as JSON format. Please read [#538](https://github.com/appium/appium-espresso-driver/issues/538) as an example usage. | e.g. `{"es": ["test data as string"], "action": "android.intent.action.MAIN", "categories": "android.intent.category.LAUNCHER", "component": "com.appium/.launcher.MainActivity", "flags": "FLAG_ACTIVITY_NEW_TASK"}` |
```
Comment (mykola-mokhnach) [inline docs/en/writing-running-appium/caps.md]: I'd rather add this option later after @rajdeepv creates more detailed documentation for it.

Comment (KazuCocoa) [reply docs/en/writing-running-appium/caps.md]: Yeah, great to add as another documentation here

File: docs/en/advanced-concepts/settings.md [hunk 3]
```diff
@@ -93,4 +93,4 @@ Settings are implemented via the following API endpoints:
...
-| `screenshotOrientation` | Adjust screenshot orientation for iOS. Appium tries to return a screenshot correctly adjusting the orientation automatically by default, but it sometimes does not work, especially landscape mode. It depends on various factors such as OS versions, model versions and real or simulator devices. This option allows you to adjust the coordinate. Defaults to `auto`.| `auto`, `portrait`, `portraitUpsideDown`, `landscapeRight` | 1.17.0+ |
+| `screenshotOrientation` | Adjust screenshot orientation for iOS. Appium tries to return a screenshot correctly adjusting the orientation automatically by default, but it sometimes does not work, especially in landscape mode. The actual screenshot orientation depends on various factors such as OS versions, model versions and whether this is a real or simulator device. This option allows you to enforce the given image orientation. Defaults to `auto`.| `auto`, `portrait`, `portraitUpsideDown`, `landscapeRight`, `landscapeLeft` | 1.17.0+ |
```
Comment (mykola-mokhnach) [inline docs/en/advanced-concepts/settings.md]: ... a screenshot and adjust its orientation properly using internal heuristics, but sometimes it does not work...

**Review by mykola-mokhnach — APPROVED** [review]

File: docs/en/writing-running-appium/caps.md (hunk 2 above)
Comment (rajdeepv) [reply docs/en/writing-running-appium/caps.md]: I have added a paragraph about this in https://github.com/appium/appium/pull/13901
Not sure how much detailed should this be, please let me know

**Pull request merged**